### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Kwaleyela-Ikafa**
in this branch we:

- Created Calculator UI
- Used React components.
- Used React props.
- Used React state

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,16 +3,10 @@ import React from 'react';
...
 root.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>
 );
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
```
Comment (GzimAsani) [inline src/index.js:18]: - [x] You have a folder created that you do not need, kindly remove it 
image

Comment (GzimAsani) [reply src/index.js:18]: - [x] Remove these images, they will not be needed 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @Kwaleyela-Ikafa        :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Your style looks good, nice job +1: 
 
 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 




 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Kwaleyela-Ikafa) [reply src/index.js:18]: Didn't see those thanks, but you would have approved and just pointed it out for me to remove them it's not something to waste a review on honestly

Comment (GzimAsani) [reply src/index.js:18]: @Kwaleyela-Ikafa sorry but they are project requirements  
image

**Review by DammyShittu — APPROVED** [review]
Hi @Kwaleyela-Ikafa,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Kwaleyela-Ikafa) [reply src/index.js:18]: I don't know how i forgot them there, appreciate the review though

**Pull request merged**